tests/operator_test.go

```diff
@@ -375,6 +375,17 @@ var _ = Describe("[Serial]Operator", func() {
 
 		patchKvInfra = func(name string, infra *v1.ComponentConfig) {
 			kv := copyOriginalKv()
+			if infra == nil {
+				kv, err := virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Get(kv.Name, &metav1.GetOptions{})
+				ExpectWithOffset(1, err).NotTo(HaveOccurred())
+				kv.Spec.Infra = infra
+				EventuallyWithOffset(1, func() error {
+					_, err := virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(kv)
+					return err
+				}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+				return
+			}
+
```

Comment:
why is this necessary? same with the patchKvWorkloads change. Maybe we should be creating an "Add" patch instead of a replace?

Reply:
The test is trying to remove the infra/workload field. With patch, it actually set it to null but that field is not nullable. I guess we could use remove?

Reply:
oh, i miss read the intent. Yeah, try remove. If that doesn't work, you could replace the entire spec field if you needed.